templates/c2s-pharmacy-finder/1/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '2'
+
+services:
+
+  c2s-frontend:
+    image: "nhsuk/connecting-to-services:${c2s_docker_image_tag}"
+    environment:
+      NODE_ENV: production

```

Comment:
Mmm - http://unix.stackexchange.com/questions/32001/what-is-m-and-how-do-i-get-rid-of-it 
This is what happens when I Copy+Paste config from @adamgraves-choices